config/postgres/initdb/scripts/create-tables-and-functions.sql

```diff
@@ -60,3 +60,21 @@ CREATE TABLE analyses (
     user_id        UUID NOT NULL REFERENCES users(user_id) ON DELETE CASCADE,
     PRIMARY KEY (client_id, case_id)
 );
+
+CREATE FUNCTION CREATE_USER( email citext, admin boolean, hashed_password text)
```

Comment:
Should the function name be lower_case?

Reply:
Also what happens if the user of the function does not pass all three parameters? I'm not familiar with SQL, but it seems `admin` is a required parameter. Can we make it optional? It should automatically default to `FALSE` if it's not specified.

Reply:
Yes, it looks like the convention is to match the format of function names and table names, which we're making lowercase where the table is created and uppercase where it is referenced, so I changed the create_user function declaration to be lowercase. 

Making the admin param optional sounds good too. Done.